lib/src/corpus.dart

```diff
@@ -220,7 +220,7 @@ class Project {
       : name = json[MetadataKeys.projectName],
         overlayPath = json[MetadataKeys.overlayPath],
         host = SourceHost.fromJson(json[MetadataKeys.host]),
-        metadata = jsonDecode(json[MetadataKeys.metadata]);
+        metadata = json[MetadataKeys.metadata];
```

Comment:
No change required, but as a side note, `MetadataKeys` seems like an odd namespace for the keys in the entry _containing_ the metadata. I expected it to be the names of the keys inside the metadata table.

Reply:
Yeah.  This needs some thinking.  Thanks for the nudge.